### ISSUE TITLE
Maybe a coding error

### ISSUE THREAD

**Issue by HouTong-s** [hssip/FashionSAP#7]
1、
in the models/model_fashion_tgir.py line 128:
fusion_feat_m = F.normalize(self.combine_text_proj(self.text_proj_m(fusion_output_m.last_hidden_state[:,0,:])), dim=-1)

I think it should be : 
fusion_feat_m = F.normalize(self.combine_text_proj_m(self.text_proj_m(fusion_output_m.last_hidden_state[:,0,:])), dim=-1)

the difference is combine_text_proj and combine_text_proj_m

2、
in the models/model_fashion_pretrain.py line 174:
symbol_feature = F.normalize(sign_feat, dim=-1)

I think it is unnecessary to use a F.normalize because there is : 
"sign_feat = F.normalize(self.combine_text_proj(self.text_proj(text_embeds[:,1,:])), dim=-1) "  in previous line 

@hssip what dou you think?
Hope it will be replied.

**Comment by hssip** [comment]
Thanks for your attention to our work and the suggestions are great.  We have fixed the bug and reduced operations in [commit](https://github.com/hssip/FashionSAP/commit/f98c70d4ff7cdbb4231c2251edbc6db20edadba6).
